cli/localci/cli/status.py

```
@@ -52,9 +54,27 @@ def status(
-    logs_dir: Path = cfg.logging.directory
-
-    # Locate the results file
+    logs_dir = Path(cfg.logging.directory)
+
+    # Prefer MCP status file when no execution-id specified
+    status_file = logs_dir / "last-status.json"
+    if not execution_id and status_file.exists():
+        try:
+            with open(status_file, encoding="utf-8") as f:
+                status_data = json.load(f)
+        except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)